erpnext/accounts/doctype/subscription_plan/subscription_plan.py

```diff
@@ -13,3 +14,15 @@ def validate(self):
 	def validate_interval_count(self):
 		if self.billing_interval_count < 1:
 			frappe.throw('Billing Interval Count cannot be less than 1')
+
+	def get_plan_rate(self, quantity=1, customer=None):
```

Comment:
Can we make this as non-class method, by sending required parameters to the function? This will eliminate multiple get_doc calls, as get_doc is much expensive than get_value.

- https://github.com/frappe/erpnext/pull/14707/files#diff-40394f921778a3645f3a27a3c6cca898R44
- https://github.com/frappe/erpnext/pull/14707/files#diff-063c1198a399addb6761f17122979710R308

Reply:
Sure I will propose a new solution